pantheonio/Pantheon
# 🚀 Chat + Subscriptions

**Steps to test:**
1. Create 2 users 
2. Login into the system with the user 1
3. Start Listening the messageReceived Subscription
4. Create a conversation
5. Send a message
6. See if your message Appears (SHOULD NOT APPEAR!)
7. Login into the system with the user 2
8. Send a message to the same chat 
9. See the subscription again now should appear a notification for the user! ❤️
